ui/src/components/Sidebar/Sidebar.tsx

```diff
@@ -16,6 +18,78 @@ import ShipName from '@/components/ShipName';
 import Avatar, { useProfileColor } from '@/components/Avatar';
 import useGroupSort from '@/logic/useGroupSort';
 import { useNotifications } from '@/notifications/useNotifications';
+import ArrowNWIcon from '../icons/ArrowNWIcon';
+import MenuIcon from '../icons/MenuIcon';
+
+export function GroupsAppMenu() {
+  const [menuOpen, setMenuOpen] = useState(false);
+  return (
+    <SidebarItem
+      div
+      className={cn(
+        menuOpen
+          ? 'bg-gray-100 text-gray-800'
+          : 'text-black hover:text-gray-800',
+        'group'
+      )}
+      icon={
+        <DropdownMenu.Root onOpenChange={() => setMenuOpen(!menuOpen)}>
+          <DropdownMenu.Trigger asChild className="appearance-none">
+            <div
+              className={cn(
+                'h-6 w-6 rounded mix-blend-multiply group-hover:bg-gray-100'
```

Comment:
this is making icon disappear in dark mode and doesn't seem to change when removing

```suggestion
                'h-6 w-6 rounded group-hover:bg-gray-100'
```